### PR TITLE
Entry points CLI 1

### PR DESCRIPTION
**Pull request by bbuzz31**
Replaces #439 which can no longer be updated due on github with `fatal: reference is not a tree: 0dfd1440e54340375ab73a982eb6439a7da8ffd1`.

I've implemented the ++proc functionality for the default delay calculations, downloadGNSS, and delays from a GUNW.

Nothing has changed for the user. They can still run `raider.py CONFIG` or `raiderDownloadGNSS.py` ...

But now with the hooks we can do other stuff with `raider.py`, like run `calcDelaysGUNW` (which is not implemented yet).
```
raider.py ++help
usage: raider.py [+h] [++process {calcDelays,downloadGNSS,calcDelaysGUNW}]

options:
  +h, ++help            show this help message and exit
  ++process {calcDelays,downloadGNSS,calcDelaysGUNW}
                        Select the HyP3 entrypoint to use (default: calcDelays)
```

```
raider.py -h
usage: calcDelays [-h] [-g] [--download_only] [customTemplateFile]

Command line interface for RAiDER processing with a configure file.Default options can be found by running: raider.py --generate_config

positional arguments:
  customTemplateFile    custom template with option settings. ignored if the default smallbaselineApp.cfg is input.

options:
  -h, --help            show this help message and exit
  -g, --generate_template
                        generate default template (if it does not exist) and exit.
  --download_only       only download a weather model.

Examples of use:
         raider.py customTemplatefile.cfg
         raider.py -g
```
Replaces #432

### CODE REVIEW

**Comment by bbuzz31** [comment]
Outstanding comments: 
https://github.com/dbekaert/RAiDER/pull/439#discussion_r1046664240
https://github.com/dbekaert/RAiDER/pull/439#discussion_r1046663814
https://github.com/dbekaert/RAiDER/pull/439#discussion_r1046654656
https://github.com/dbekaert/RAiDER/pull/439#discussion_r1046663814 (Note default is now HRRR)

**Comment by bbuzz31** [comment]
pinning ISCE version fixed ISCE/GDAL compatability (#425) and closes #433

File: tools/RAiDER/aria/calcGUNW.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+def main(args):
+    """ Calculate  interferometric phase delay at dates in ARIA standard product (GUNW) """
```
Comment (dbekaert) [inline tools/RAiDER/aria/calcGUNW.py:2]: Calculate interferometric phase delay for an ARIA standard product (GUNW)

**Review by dbekaert — APPROVED** [review]
LGTM

**Pull request merged**